doc-content/enterprise-only/fuse/installing-on-fuse-eap-proc.adoc

```diff
@@ -1,18 +1,20 @@
 [id='installing-on-fuse-eap-proc']
 = Installing {FUSE} on {EAP_LONG}
-Install {FUSE_LONG} 7.6 on {EAP} 7.2.3 to use with {PRODUCT} to deploy integrated services where required.
+Install {FUSE_LONG} {FUSE_VERSION} on {EAP} 7.2.7 to use with {PRODUCT} to deploy integrated services where required.
```

Comment:
Can we introduce an attribute for the EAP version that is specified here (7.2.7) as well similar to the Fuse version?

Reply:
We could introduce a new attribute to capture the specific EAP 7.2 version, it would be a good idea if there would be a version mismatch going forward, but this will be deprecated once RHPAM/RHDM 7.9 is re-certified with Fuse 7.8 on EAP 7.3.

Reply:
Okay, got it